### PR TITLE
LPFG-145 modules in search results

### PR DESCRIPTION
**Pull request by smoga-git**

### CODE REVIEW

File: src/ui/component/DisciteModule.html [hunk 1]
```diff
@@ -1,24 +1,43 @@
 <div class="discite__item u-clearfix discite__item--module">
-    <div class="discite__main discite__main--module">
-        <h3 class="heading bold-small heading--text">
-            {{module.title}}
-        </h3>
+    <div class="discite__main discite__main--module--{{modifier || ''}}">
```
Comment (jenbutongit) [inline src/ui/component/DisciteModule.html]: in BEM there shouldn't be any cases where you have a modifier modifier. I've created an element  .discite__counter. Could you put this in there? refer to `DisciteCourse.html` for structure.

File: src/ui/assets/styles/components/_discite.scss [hunk 2]
```diff
@@ -38,6 +38,12 @@ $module: 'discite';
 		@include media(desktop) {
 			padding-right: 300px;
 		}
+
+		@include m(module) {
```
Comment (jenbutongit) [inline src/ui/assets/styles/components/_discite.scss]: If you need to you can use discite__counter and add the modifier `@include m(module)` in there, and get rid of the nested modifier

File: src/ui/component/DisciteModule.html (hunk 1 above)
Comment (smoga-git) [reply src/ui/component/DisciteModule.html]: done

File: src/ui/assets/styles/components/_discite.scss (hunk 2 above)
Comment (smoga-git) [reply src/ui/assets/styles/components/_discite.scss]: done

**Pull request merged**